packages/cacti-copm-core/Makefile

```diff
@@ -0,0 +1,194 @@
+WORKSPACE ?= $(shell realpath `pwd`/../../)
+DOCKER_IMAGES=hyperledger/fabric-orderer hyperledger/fabric-peer hyperledger/fabric-ca ghcr.io/hyperledger/cacti-weaver-iin-agent ghcr.io/hyperledger/cacti-weaver-relay-server ghcr.io/hyperledger/cacti-weaver-driver-fabric corda/community:4.12-zulu-openjdk ghcr.io/hyperledger/cacti-weaver-driver-corda copm-corda
+
+
+# DL-NETWORK ---------------------------------------------------
+
+fabric-dl: 
+	export GO111MODULE=auto
+	echo "making fabric DL network"
+	cd $(WORKSPACE)/weaver/tests/network-setups/fabric/dev && \
+	make start-interop-local CHAINCODE_NAME=$(chaincode) PROFILE='2-nodes'
+
+corda-pledge-dl:
+	cd $(WORKSPACE)/weaver/tests/network-setups/corda && \
+	./scripts/get-cordapps.sh simple local && \
+	make start-local
+
+corda-lock-dl:
+	cd $(WORKSPACE)/weaver/tests/network-setups/corda && \
+	./scripts/get-cordapps.sh  simple local  && \
+	make start-network1-local PROFILE=3-nodes
+
+# DRIVERS -----------------------------------------------------
+
+fabric1-driver:
+	cd $(WORKSPACE)/weaver/core/drivers/fabric-driver && \
+	sed -i "s#<PATH-TO-WEAVER>#$(WORKSPACE)/weaver#g" docker-testnet-envs/.env.n1 &&\
+	make deploy COMPOSE_ARG='--env-file docker-testnet-envs/.env.n1' NETWORK_NAME=network1 
+
+fabric2-driver:
+	cd $(WORKSPACE)/weaver/core/drivers/fabric-driver && \
+	sed -i "s#<PATH-TO-WEAVER>#$(WORKSPACE)/weaver#g" docker-testnet-envs/.env.n2 && \
+	make deploy COMPOSE_ARG='--env-file docker-testnet-envs/.env.n2' NETWORK_NAME=network2
+
+corda1-driver:
+	echo "starting corda driver 1"
+	cd $(WORKSPACE)/weaver/core/drivers/corda-driver && \
+	make deploy COMPOSE_ARG='--env-file docker-testnet-envs/.env.corda'
+
+corda2-driver:
+	echo "starting corda driver 2"
+	cd $(WORKSPACE)/weaver/core/drivers/corda-driver && \
+    make deploy COMPOSE_ARG='--env-file docker-testnet-envs/.env.corda2'
+
+
+# RELAYS ---------------------------------------------------------
+
+common-relay-config:
+	cd $(WORKSPACE)/weaver/core/relay && \
+	make convert-compose-method2
+
+corda-relay-config:
+	cd $(WORKSPACE)/weaver/core/relay &&\
+    make protos-local &&\
+    cargo update -p nom &&\
+    cargo update -p lexical-core
+
+corda1-relay:
+	echo "starting corda relay 1"
+	cd $(WORKSPACE)/weaver/core/relay &&\
+	make start-server COMPOSE_ARG='--env-file docker/testnet-envs/.env.corda' 
+
+corda2-relay:
+	echo "starting corda relay 2"
+	cd $(WORKSPACE)/weaver/core/relay &&\
+    make start-server COMPOSE_ARG='--env-file docker/testnet-envs/.env.corda2'
+
+fabric1-relay:
+	cd $(WORKSPACE)/weaver/core/relay && \
+	make start-server COMPOSE_ARG='--env-file docker/testnet-envs/.env.n1' 
+
+fabric2-relay:
+	cd $(WORKSPACE)/weaver/core/relay && \
+	make convert-compose-method2 && \
+	make start-server COMPOSE_ARG='--env-file docker/testnet-envs/.env.n2' 
+
+# IIN --------------------------------------------------------------------------------------
```

Comment:
@sandeepnRES I think we don't need the `iin` agents for scenarios involving Corda, right? They won't do anything useful.

Reply:
@jenniferlianne Is this used for Fabric-Fabric testing too, or just ones that involve Corda networks?

Reply:
This file is used for testing all combinations.  I will move it from cacti-copm-core to cacti-copm-test as it references the weaver samples.